File: src/main/java/com/trade/securities/domain/ElwTradingInfo.java
package com.trade.securities.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ElwTradingInfo {
    private String basDd; // 기준일자
    private String isuCd; // 종목코드
    private String isuNm; // 종목명
    private String tddClsprc; // 종가
    private String cmpprevddPrc; // 대비
    private String tddOpnprc; // 시가
    private String tddHgprc; // 고가
    private String tddLwprc; // 저가
    private String accTrdvol; // 거래량
    private String accTrdval; // 거래대금
    private String mktcap; // 시가총액
    private String listShrs; // 상장증권수
    private String ulyNm; // 기초자산 지산명
    private String ulyPrc; // 기초자산 종가
    private String cmpprevddPrcUly; // 기초자산 대비
    private String flucRtUly; // 기초자산 등락률
}

File: src/main/kotlin/com/trade/securities/domain/ElwTradingInfo.kt
package com.trade.securities.domain

data class ElwTradingInfo(
    val basDd: String? = null, // 기준일자
    val isuCd: String? = null, // 종목코드
    val isuNm: String? = null, // 종목명
    val tddClsprc: String? = null, // 종가
    val cmpprevddPrc: String? = null, // 대비
    val tddOpnprc: String? = null, // 시가
    val tddHgprc: String? = null, // 고가
    val tddLwprc: String? = null, // 저가
    val accTrdvol: String? = null, // 거래량
    val accTrdval: String? = null, // 거래대금
    val mktcap: String? = null, // 시가총액
    val listShrs: String? = null, // 상장증권수
    val ulyNm: String? = null, // 기초자산 지산명
    val ulyPrc: String? = null, // 기초자산 종가
    val cmpprevddPrcUly: String? = null, // 기초자산 대비
    val flucRtUly: String? = null // 기초자산 등락률
)
